app/graphql.js

```diff
@@ -15,6 +18,16 @@ const authMiddleware = {
     req.options.headers.authorization = `Bearer ${getJwtToken()}`;
     next();
   },
+  applyAfterware({ response }, next) {
+    response.clone().json().then((body) => {
+      if (body.errors && body.errors.some((e) => e.status === 401 || e.status === 403)) {
+        deleteJwtToken();
+        browserHistory.replace(loginPage(browserHistory.getCurrentLocation().pathname));
```

Comment:
Jeżeli pytasz czy to jest świadome to tak, a jeżeli o to czy jestem pewnie czy chcemy w ten sposób to nie. Uznałem że w takiej sytuacji zapytanie jest już obsłużone i nie trzeba wołać next(). Co nie znaczy że jestem przekonany że tak powinniśmy zrobić.